src/sagas/networkSettings.js

```diff
@@ -282,6 +283,8 @@ export function* persistNetworkSettings(action) {
     return;
   }
 
+  yield put(networkChanged());
```

Comment:
Why do you need this here? Maybe add a comment

Reply:
We use this to clear sessions on reown. Added the comment